site/docs/getting-started/v3/microsoft-testing-platform.md

```diff
@@ -22,8 +22,7 @@ The xUnit.net integration with Microsoft Testing Platform comes at three levels:
 2. You can run tests with the new Microsoft Testing Platform integrated `dotnet test` ([&#x1F517;](#enabling-the-dotnet-test-experience))
 3. You can run tests with the new Microsoft Testing Platform integrated Test Explorer ([&#x1F517;](#enabling-the-test-explorer-experience))
 
-Unlike our support for VSTest, our support for Microsoft Testing Platform is built natively into xUnit.net v3. If you want to rely solely on Microsoft Testing Platform support, you can remove the package references to `xunit.runner.visualstudio` and `Microsoft.NET.Test.Sdk`. However, for backward compatibility reasons, we recommend you leave these in place, because as of the writing of this document, third party runners (like Resharper/Rider, CodeRush, and Visual Studio Code) still rely on VSTest to be able to run xUnit.net tests. Once all runners can support Microsoft Testing Platform, then we'll be able to deprecate `xunit.runner.visualstudio`. Supporting VSTest is separate from (and does not interfere with) our support for Microsoft Testing Platform.
-
+Unlike our support for VSTest, our support for Microsoft Testing Platform is built natively into xUnit.net v3. If you want to rely solely on Microsoft Testing Platform support, you can remove the package references to `xunit.runner.visualstudio` and `Microsoft.NET.Test.Sdk`. However, for backward compatibility reasons, we recommend you leave these in place, because as of the writing of this document, third party runners (like Resharper and CodeRush) still rely on VSTest to be able to run xUnit.net tests. Once all runners can support Microsoft Testing Platform, then we'll be able to deprecate `xunit.runner.visualstudio`. Supporting VSTest is separate from (and does not interfere with) our support for Microsoft Testing Platform.
```

Comment:
Rider and DevKit support MTP now.
I'm not really sure about ReSharper though.
CodeRush is likely not supporting it.

Reply:
Thanks for the update!

I'd be surprised if it was added only to Rider and not Resharper, as I thought they shared this code base (though my information here is wildly out of date 😂).